.github/workflows/documentation.yml

```diff
@@ -0,0 +1,34 @@
+name: Documentation
+on:
+  push:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: build
+          auto-update-conda: true
+
+      - name: Install requirements
+        run: |
+          conda install -n build -c conda-forge --file requirements/docs.txt --quiet --yes
+
+      - name: Install the package
+        run: python -m pip install .
+
+      - name: Build documents
+        run: make -C doc html
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          publish_branch: gh-pages
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./doc/build/html
+          force_orphan: true
```

Comment:
delete blank line at end (preferrably set up your PyCharm to do that automatically in settings)